python/py-scikit-rf/Portfile

```diff
@@ -24,9 +24,10 @@ long_description    scikit-rf (aka skrf) is an Open Source, BSD-licensed \
 python.versions     37 38
 
 if {${name} ne ${subport}} {
-    depends_build-append \
+    depends_lib-append \
         port:py${python.version}-numpy \
-        port:py${python.version}-scipy \
+        port:py${python.version}-scipy
+    depends_build-append \
         port:py${python.version}-setuptools
     livecheck.type  none
```

Comment:
please keep the order of dependencies in the way they're needed, so `build` first and then `lib`; also aligning it with the rest of the Portfile would be good.